src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PeopleList } from '../PeopleList';
+
+type Props = {
+  people: Person[]
+};
+
+export const PeopleTable = ({ people }: Props) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className={classNames('table',
+        'is-striped',
+        'is-hoverable',
+        'is-narrow',
+        'is-fullwidth')}
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
The names of the columns can be written in an array and re-rendered through the `map`.